### PR TITLE
DOCS-1489: Change Python virtual environment string

### PR DESCRIPTION
**Pull request by JessamyT**

### CODE REVIEW

File: docs/build/program/python-venv.md [hunk 1]
```diff
@@ -48,16 +48,16 @@ cd viam-python
...
 ```
 
-Now, `(viam-env)` prepends the commands in your terminal window to indicate the Python packages being used are from this particular environment.
+Now, `.venv` prepends the commands in your terminal window to indicate the Python packages being used are from this particular environment.
 You can exit this environment by running `deactivate`.
 
 ## Install Viam
 
-Inside the activated `viam-env` python environment, you can now install the Viam Python SDK:
+Inside the activated `.venv Python environment, you can now install the Viam Python SDK:
```
Comment (JessamyT) [inline docs/build/program/python-venv.md:60]:
```suggestion
Inside the activated `.venv` Python environment, you can now install the Viam Python SDK:
```

**Review by npentrel — CHANGES_REQUESTED** [review]
This is good but the ticket indicates they changed the in app code sample page which we'll need to update [here](https://docs.viam.com/build/program/#requirements).

Screenshot 2024-03-15 at 12 28 35

**Review by hexbabe — APPROVED** [review]

**Comment by JessamyT** [comment]
@npentrel Are you saying that on the #requirements section there, instead of just linking to setting up a virtual environment, we should actually include the commands to do so on that Program page as well? Can do.

**Comment by JessamyT** [comment]
Should we update https://python.viam.dev/ too?

**Review by npentrel — APPROVED** [review]

**Pull request merged**